packages/frontend/core/src/modules/peek-view/view/doc-peek-view.css.ts

```diff
@@ -7,3 +8,12 @@ export const editor = style({
   },
   minHeight: '100%',
 });
+
+export const affineDocViewport = style({
+  display: 'flex',
+  flexDirection: 'column',
+  userSelect: 'none',
```

Comment:
What is the purpose of using `userSelect: 'none'`?

Reply:
sorry, I cannot recall why it is added

Reply:
Never mind, I deleted this line so that the text can be selected in read-only mode